### PR TITLE
refactor(select): new simple and multi select UX and Accessibility improvements

### PR DESCRIPTION
**Pull request by Supamiu**
## Description

This PR rewrites a lot of the template, interactions and accessibility for the `lu-simple-select` and `lu-multi-select` components.

Notable changes:

- Search is now in the input itself and not in the panel anymore
- All useful `aria-*` attribute are now handled and connected
- The general design of the select inputs has changed
- New `lu-simple-select-field` and `lu-multi-select-field` components to use in forms (includes label, description, state handling, etc)
- You can now group options in display ( #2379 )
- New `luDisplayerInput` directive to connect `aria-*` and every panel interactions to your custom-made displayer in multi-select

-----

No changes have been applied to existing inputs, only new things added and none of them are required.

If you want search to be included in your select input, you'll have to implement it yourself like you can see in "With Clue" stories, implementing a basic search that uses either option value or option.name value is possible but it'll be done only if the need for it is present.

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2467/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/10925/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2467/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/10984/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2467/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/10995/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2467/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/10996/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2467/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/11011/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2467/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/11013/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2467/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/11017/ui-diff)

**Review by jeremie-lucca — DISMISSED** [review]

**Review by vvalentin-lucca — APPROVED** [review]

**Review by GuillaumeNury — DISMISSED** [review]

File: packages/ng/forms/simple-select-field/simple-select-field.component.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+@Component({
+	selector: 'lu-simple-select-field',
+	standalone: true,
+	imports: [FormFieldComponent, InputDirective, ReactiveFormsModule, LuSimpleSelectInputComponent],
+	templateUrl: './simple-select-field.component.html',
+	hostDirectives: [NoopValueAccessorDirective],
+	changeDetection: ChangeDetectionStrategy.OnPush,
+	encapsulation: ViewEncapsulation.None,
+})
+export class SimpleSelectFieldComponent<TOption> extends AbstractFieldComponent {
```
[the remark this answers is not in the document]
Comment (Supamiu) [reply packages/ng/forms/simple-select-field/simple-select-field.component.ts:21]: Fixed by #2502

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2467/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/11671/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2467/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/11675/ui-diff)

**Pull request merged**